libs/route-config/README.md

```diff
@@ -182,3 +227,26 @@ In this case your example router config can look like this:
 })
 export class AppModule {}
 ```
+
+#### Providing default value globally
+
+Both `*tdRouteData` and `RouteConfigService` allow for providing a default value in case route data doesn't provide a certain property. It is also possible to provide a default value globally by providing `ROUTE_DATA_DEFAULT_VALUE` injection token when configuring the root module:
+
+```ts
+@NgModule({
+  /* other module props  */
+  imports: [RouteConfigModule.forRoot() /* other modules */],
+  provide: [
+    {
+      provide: ROUTE_DATA_DEFAULT_VALUE,
+      useValue: {
+        title: 'Injected Default Title',
+        someDefaultParam: 'Some other default param',
+      },
+    },
+  ],
+})
+export class AppModule {}
+```
+
+This way we don't need to provide default value each time we use `*tdRouteData` or `RouteConfigService`. However, if a different default value is necessary it still can be provided - and it will overwrite the value injected with a token.
```

Comment:
I'd add that this way it can be overriden on component level as well

```
@Component({
   // ...
   providers: [ /* the value provided here will be the default down in the DOM tree */]
})
```

Reply:
done